web/packages/teleport/src/JoinTokens/JoinTokenGithubForm.test.tsx

```diff
@@ -38,28 +39,21 @@ const populateRuleFieldTest =
     value: string
   ) =>
   async () => {
-    const state = baseState();
-    const onUpdate = jest.fn();
+    const { user, onUpdate } = renderComponent();
 
-    render(
-      <JoinTokenGithubForm
-        tokenState={state}
-        onUpdateState={onUpdate}
-        readonly={false}
-      />,
-      { wrapper: Wrapper }
-    );
+    const input = screen.getByPlaceholderText(placeholer);
+    expect(input).toBeInTheDocument();
```

Comment:
Is it necessary to call `toBeInTheDocument` when `screen.getByPlaceholderText` is already going to fail if the input is not found? It doesn't look like [`toBeInTheDocument` does anything extra](https://github.com/testing-library/jest-dom/blob/0ff8904ff4683d676ff70ab68b7f08465c44d0d0/src/to-be-in-the-document.js).